### PR TITLE
[Docs] Fix `sendGAEvent` example in `@next/third-parties`

### PR DESCRIPTION
**Pull request by tschoffelen**
Fixes the usage example for `sendGAEvent()` exposed by `@next/third-parties`.

This pushes to GTAG's dataLayer, which means it needs to follow the structure set out in [the Google docs here](https://developers.google.com/analytics/devguides/collection/ga4/event-parameters?client_type=gtag#set-up-event-parameters):

```js
gtag('event', 'screen_view', {
  'app_name': 'myAppName',
  'screen_name': 'Home'
});
```

This also matches with the function signature, which looks like this: `function sendGAEvent(..._args: Object[])`

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f8915f1411a12e8e8d7bfd57119aa60af8b9e054

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by samcx** [comment]
@tschoffelen Is this Google docs site link down for you?

**Comment by tschoffelen** [comment]
> @tschoffelen Is this Google docs site link down for you?

Hmm no it works for me, let me try again:

https://developers.google.com/analytics/devguides/collection/ga4/event-parameters?client_type=gtag

**Comment by samcx** [comment]
@tschoffelen Very strange 🤔 

CleanShot 2024-05-14 at 10 38 47@2x

**Comment by samcx** [comment]
@tschoffelen The <picture data-single-emoji=":docs:" title=":docs:">:docs:</picture> have already been <picture data-single-emoji=":fixed:" title=":fixed:">:fixed:</picture> (https://rc.nextjs.org/docs/app/building-your-application/optimizing/third-party-libraries), so will be closing this!
